### PR TITLE
#1017 Preserve whitespace in between tags

### PR DESCRIPTION
**Pull request by imranazad**
https://github.com/fsharp/FSharp.Data/issues/1017#issuecomment-380422441

### CODE REVIEW

File: src/Xml/XmlRuntime.fs [hunk 1]
```diff
@@ -86,7 +86,7 @@ type XmlElement =
   static member Create(reader:TextReader) =    
     use reader = reader
     let text = reader.ReadToEnd()
-    let element = XDocument.Parse(text).Root 
+    let element = XDocument.Parse(text, LoadOptions.PreserveWhitespace).Root
```
Comment (ovatsus) [inline src/Xml/XmlRuntime.fs]: You also need to change CreateList

**Comment by ovatsus** [comment]
Can you add tests?

File: src/Xml/XmlRuntime.fs (hunk 1 above)
Comment (imranazad) [reply src/Xml/XmlRuntime.fs]: Hey @ovatsus Sorry for the delay. Thanks I've changed that and added tests now.

**Comment by ovatsus** [comment]
Didn't merge it yet because both AppVeyor and Travis failed and I didn't have a chance yet to look why it failed

**Comment by imranazad** [comment]
@ovatsus I've fixed the failing tests could you please complete the code review when you get a moment and merge it in. We're quite eager to use the latest package.

**Comment by chunter** [comment]
@ovatsus Would it be possible to check and merge these changes in as we really need this fix and also would like to use the latest code.

**Comment by ovatsus** [comment]
Sorry, haven't had time to look at this lately, I'll review and merge this this weekend

**Comment by ovatsus** [comment]
You had missed one place, fixed it

**Comment by ovatsus** [comment]
This is now released as part of 3.0.0-beta4. Sorry for taking so long

**Comment by imranazad** [comment]
@ovatsus That's brilliant, many thanks.

**Pull request merged**